classes/mender-image.bbclass

```diff
@@ -9,4 +9,4 @@ inherit mender-image-buildinfo
 
 #Make sure we are creating sdimg with all needed partitioning.
 IMAGE_CLASSES += "mender-sdimg"
-IMAGE_FSTYPES ?= "tar.gz ext3 sdimg"
+IMAGE_FSTYPES_append = " sdimg"
```

Comment:
Hmm, one thing I didn't consider is that this will make it impossible to *not* build the image. This may in fact be quite common, since the subsequent updates only need the rootfs, not the sdimg. Any thoughts on this?